### PR TITLE
update license header to 2021

### PR DESCRIPTION
**Pull request by RKORavi**
#### Describe Bug or Feature
update license header to 2021

#### Describe Your Change
Updated license header from 2020 to 2021

#### Link to Design Doc
NA

### CODE REVIEW

File: src/modules/python/pbs/__init__.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 """
 
 /*
```
Comment (Shrini-h) [inline src/modules/python/pbs/__init__.py:4]: This C style comment syntax may be deleted

File: src/modules/python/setup.py [hunk 2]
```diff
@@ -2,7 +2,7 @@
 """
 
 /*
```
Comment (Shrini-h) [inline src/modules/python/setup.py:4]: This C style comment syntax may be deleted

**Comment by RKORavi** [comment]
[root@simvm2 master_lic_fix]# python pbspro-extensions/src/tools/pbs_copyright_check.py /home/pbsadmin/gitRemote/master_lic_fix
[root@simvm2 master_lic_fix]# more outdated.txt 
/home/pbsadmin/gitRemote/master_lic_fix/openpbs/src/include/pbs_v1_module_common.i
/home/pbsadmin/gitRemote/master_lic_fix/openpbs/src/lib/Libifl/pbs_ifl.i
/home/pbsadmin/gitRemote/master_lic_fix/pbspro-extensions/src/simulator/src/sim.conf.in

**Review by Shrini-h — APPROVED** [review]
LGTM , Thank you for making the changes

**Review by sakshamgarg — APPROVED** [review]

**Review by prakashcv13 — APPROVED** [review]

**Pull request merged**